explorer/src/lib/components/blocks-card/BlocksCard.css

```diff
@@ -0,0 +1,5 @@
+.blocks-card__table,
+.blocks-card__list {
+  max-height: 600px;
```

Comment:
We'll revise layout things and scrolling behaviours later, but in the meanwhile can these cases be expressed as a relative measure instead? Like in `dvh` for example?